plugins/dell-dock/fu-dell-dock-i2c-ec.c

```diff
@@ -106,6 +118,7 @@ typedef enum {
 	MODULE_TYPE_130_UNIVERSAL,
 	MODULE_TYPE_240_TRIN,
 	MODULE_TYPE_210_DUAL,
+	MODULE_TYPE_130_USB4,
```

Comment:
There is no differentiation for greater or lesser voltages connected to USB4 module?

Reply:
This ID just indicates the type of cable and the wattage designation just identifies the maximum wattage that is supported by the cable.